src/page/Auth/Signup/component/ErrorMessage/index.tsx

```diff
@@ -2,14 +2,16 @@ import { ReactComponent as Warn } from 'assets/svg/auth/warning.svg';
 import styles from './ErrorMessage.module.scss';
 
 interface ErrorMessageProps {
-  messages: (string | undefined)[]
+  messages?: (string | undefined)[]
+  message?:string
 }
 
-export default function ErrorMessage({ messages }:ErrorMessageProps) {
+export default function ErrorMessage({ messages, message }:ErrorMessageProps) {
```

Comment:
조금 더 간단히 타입을 명시하면 요렇게도 작성할 수 있을것같아요
```suggestion
export default function ErrorMessage({ errorMessage }: string | string[]) {
return (
    <div className={styles.warn}>
      <Warn />
      <span className={styles['warn--phrase']}>
        {{typeof errorMessage === 'string' ? errorMessage : errorMessage [0]}
      </span>
    </div>
```